### ISSUE TITLE
mobile.client.acct

### ISSUE THREAD

**Issue by DavidJSGardner** [polserver/polserver#461]
```
acct 	Integer 	Account Reference (if not NPC) (deprecated, will be removed in next release, use Client instead)
```

So using this crashes the shard.
`mobile.client.acct`

While I did not see the acct member in the entry for the Client object in the docs, I figured this was just an innocent omission, so I figured it is probably there since the docs direct me to use the client obj for fetching the character's acct member. If the plan is to deprecate `mobile.acct` as the docs say, I would rather write code reflecting this going forward so that I don't have to later or someone else doesn't have to later go through and fix the stuff listed in the docs (which by the way is more than a few, `ip` just to name one) as to be deprecated in the next release.

At any rate, `mobile.client.acct` is crashing the shard and it shouldn't be.

**Comment by DavidJSGardner** [comment]
Consequently I have also just tried `mobile.client.name` which also crashes the shard.

Looking at past issues I have run into, I had a similar issue with:
[https://github.com/polserver/polserver/issues/451#issue-1105536218]mobile.client.connected(url)

I am not sure what this might tell you about the client object but clearly something is very wrong with trying to access info from it if you are trying to access members that do not exist on the object would be my guess.

**Comment by turleypol** [comment]
Should be fixed with #462
